### ISSUE TITLE
svg icons and language switch

### ISSUE THREAD

**Issue by millianapia** [equinor/energyvision#3152]
• Svg-icons shouldn’t have aria-hidden (because they give more information).
• Language – screen readers currently read “switch to English” when you are on an English page. Should be "aria-selected=true", if possible, or aria-current=
